packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx

```diff
@@ -190,14 +196,20 @@ export const MainAppointmentTableColumns = (
       appointments.filter((a) => a.booked).length,
   },
   {
+    label: "",
     cellRenderer: ({ appointments }) => {
-      return (
+      var visable =
         !showOnlyRecentChanges &&
         appointments.find(
           (a) => a.recentChangeType || a.recentChangeType === 0
-        ) && <Chip colorScheme={ChipColorScheme.New} label="חדש" />
+        );
+
+      return (
+        <div style={{ visibility: visable ? "visible" : "hidden" }}>
```

Comment:
I don't like the inline styling.

You could return null if visible is false

Reply:
I did that because in that way it still takes the place and the aligning remains the same..

Reply:
You can return an empty div

Reply:
but the width of the empty div wont be the same and it will cause missalignment.

Reply:
ok